pypesto/optimize/ess/ess.py

```diff
@@ -189,24 +183,12 @@ def _initialize(self):
         self.evaluator: Optional[FunctionEvaluator] = None
         self.starttime: Optional[float] = None
 
-    def minimize(
+    def _initialize_minimize(
         self,
         problem: Problem = None,
         startpoint_method: StartpointMethod = None,
         refset: Optional[RefSet] = None,
-    ) -> pypesto.Result:
-        """Minimize the given objective.
-
-        Parameters
-        ----------
-        problem:
-            Problem to run ESS on.
-        startpoint_method:
-            Method for choosing starting points.
-            **Deprecated. Use ``problem.startpoint_method`` instead.**
-        refset:
-            The initial RefSet or ``None`` to auto-generate.
-        """
+    ):
```

Comment:
A short doctoring would still be nice, even if it is an internal `_foo` function.

Reply:
or rather a hint that doc is equivalent to `minimize`?